lib/rspec/rails/feature_check.rb

```diff
@@ -39,10 +39,6 @@ def has_action_mailbox?
         defined?(::ActionMailbox)
       end
 
-      def ruby_3_1?
-        RUBY_VERSION >= "3.1"
-      end
-
```

Comment:
If we want this it should be in RSpec Support, its a later improvement that could be made once 3.11 is released.